modules/special-rules/pages/barrage.adoc

```diff
@@ -4,29 +4,48 @@ These weapons can saturate a wide area of the battlefield with intense salvoes o
 
 ---
 
-The more closely packed the units of the target detachment, the more effective a {barrage} weapon is likely to be against it.
+TIP: The more closely packed the units of the target detachment, the more effective {barrage} weapons are likely to be against it.
 This is simply because the barrage will have more target units within its area of effect.
 
-When you shoot with a unit that has a {barrage} weapon:
+//+[IJW editing note: I've changed this to plural as you almost always have multiple barrage weapons being resolved at once.]+
 
-. Take the Barrage template and place it over the target detachment.
-Place it such that it is over as many enemy units as possible.
+When you shoot with a detachment that includes units with {barrage} weapons:
+
+. Place a single Barrage template over the target detachment, no matter how many {barrage} units in your detachment are shooting at them.
+. Place the template so that it:
+* is completely within range and line of fire of all units that contribute to the {barrage}
+* covers as many units as possible from the target detachment.
```

Comment:
I'm toying with the idea of something like this, given that I've added `barrage-templates.adoc` to the main rules (and will reference it via 'related information'):
```suggestion
. Place a Barrage template over the target detachment.
Place it in the standard way, but with these additional considerations:
* Place only a single template, no matter how many {barrage} units in your detachment contribute to the attack.
* Place the template so that it is completely within range and line of fire of *all* of the {barrage} units that contribute to the attack.
You may opt to have some units *not* contribute to the attack, but you may not make any separate attack with them.
```
It's that last point that bothers me. I'm thinking of the FAQ of course (https://thehobby.zone/resources/e40k-compendium/Content/More/QuestionsAndAnswers/QA_Barrage.htm). But, given that a player can usually split fire with a detachment, shouldn't they also be able to split fire with {barrage} units?

Reply:
The last point would specifically contradict the rules for splitting fire. So yes, it should bother you. ;-)

I assume you're referring to '(you can leave some units out if you want, but they can't fire a separate barrage)' in the linked page? That's not relevant to splitting fire, because the FAQ entry repeatedly refers to 'the target detachment'.

Also, '* covers as many units as possible from the target detachment' needs to be included in some form, as it's an important restriction.

Reply:
>The last point would specifically contradict the rules for splitting fire. So yes, it should bother you. ;-)

Yes -- that's my point, I wasn't sure how to interpret was I was reading, but I suspected that there was a problem in what I'd written. :-D

>I assume you're referring to '(you can leave some units out if you want, but they can't fire a separate barrage)' in the linked page?

Yes, exactly.

>I don't think that's relevant to splitting fire, because the FAQ entry repeatedly refers to 'the target detachment'.

Hmmm. I think I see what you mean. To my mind the wording makes it ambiguous, or at best too easy to stumble over like I just did -- I suppose it would've been a bit better if they had said "but they can't fire a separate barrage at the same target detachment"? That would've left the way more clear to infer that you can still split fire with these units if there is another target detachment with units that are within range and LoF.

>Also, '* covers as many units as possible from the target detachment' needs to be included in some form, as it's an important restriction.

Oops -- quite right. My roughing out for this conversation was a little too rough :-D

How about something like this (could perhaps be a bit tidier, but I want to check that the sense works first):

```suggestion
. Place a Barrage template over the target detachment.
Place it in the standard way, but with these additional considerations:
* Place only a single template, no matter how many {barrage} units in your detachment contribute to the attack.
* Place the template so that it is completely within range and line of fire of *all* of the {barrage} units that contribute to the attack.
You may opt to have some units *not* contribute to the attack, but you may not make any separate attack with them against the same target detachment (you may still split fire with these units if there are other suitable target detachments).
* Cover as many units in the target detachment as possible.
```

Reply:
> How about something like this (could perhaps be a bit tidier, but I want to check that the sense works first):

I think that works.

Reply:
While I'm thinking about it, I need to start a discussion on FB about how to handle 'intermingled' detachments and Barrages. But first some van repairs...